src/wp-includes/css/admin-bar.css

```
@@ -1,3 +1,7 @@
+.admin-bar {
+	--wp-admin-bar-height: 32px;
```

We've had [some discussions on variable naming](https://make.wordpress.org/core/2021/01/29/introducing-css-custom-properties/#comment-40631), and I think have landed on a pattern like `--wp-admin--[location]--[property]`. Using that format, this property should be:

```suggestion
	--wp-admin--admin-bar--height: 32px;
```